config/default.yml

```
@@ -129,6 +129,12 @@ Rails/ActiveSupportAliases:
   Enabled: true
   VersionAdded: '0.48'
 
+Rails/ActiveSupportOnLoad:
+  Description: 'Use `ActiveSupport.on_load(...)` to patch Rails core classes.'
+  StyleGuide: 'https://rails.rubystyle.guide/#hash-conditions'
+  Enabled: 'pending'
```

Can you add `Reference` URL?

```suggestion
  Reference: 'https://api.rubyonrails.org/classes/ActiveSupport/LazyLoadHooks.html'
  Enabled: 'pending'
```